### ISSUE TITLE
Loss increase after resuming training apex deepspeed

### ISSUE THREAD

**Issue by robvanvolt** [lucidrains/DALLE-pytorch#314]
Hey there, I don't know why the loss increases after resuming a two epochs training.

**First two epochs:**
https://wandb.ai/robvanvolt/dalle_train_transformer/runs/3bwiteds?workspace=user-robvanvolt
`train_dalle.py --local_rank=0 --taming --amp --image_text_folder ../dataset --truncate_captions --batch=40 --deepspeed`
<img width="417" alt="Bildschirmfoto 2021-06-20 um 12 07 21" src="https://user-images.githubusercontent.com/65366998/122670009-15869680-d1c0-11eb-97cd-8e9226c0d915.png">

**Resuming after two epochs:**
https://wandb.ai/robvanvolt/dalle_train_transformer/runs/1h7dlezt?workspace=user-robvanvolt
`train_dalle.py --local_rank=0 --taming --amp --image_text_folder ../dataset --dalle_path ./dalle.pt --truncate_captions --batch=40`
<img width="419" alt="Bildschirmfoto 2021-06-20 um 11 54 26" src="https://user-images.githubusercontent.com/65366998/122670014-1ae3e100-d1c0-11eb-9fcc-4c46ccc0d0fb.png">

Is there a systematic error I cannot identify? Do I have to add other parameters / the checkpoint path (I thought this is done automatically:P)

**edit:** Maybe I've just forgotten the --deepspeed tag on my second training? >.<*
**edit2:** Nevermind, it was just the --deepspeed flag I forgot to add.. ^^'

**Comment by robvanvolt** [comment]
Forgot --deepspeed flag...
